README.md

```diff
@@ -55,9 +55,9 @@ The backlinks calculation is costly to your computers, so it is recommended to t
 $ ./main.py -l search {URI-R} -e cdGetBacklinks
 ```
 
-## How to add your module:
+## How to add your module
 
-Name your module main script as cdGet\<Module name\>.py
+Name your module main script as cdGet\<Module name\>.py. If you module relies on extra code in a different folder it is alright to bring this directory into the modules directory, the carbon tool will ignore subfolders while loading. If the extra code is not in a subfolder, after copying it to the ./modules folder, add the file names into the config file, under 'SystemUtility' field.
```

Comment:
Correct some typos and improve highlights here:

* module main => module's main
* cdGet\<Module name\>.py => `cdGet<ModuleName>.py`
* you module => your module
* carbon tool => CarbonDate library
* while loading => while loading modules
* ./modules folder => `./modules` folder
* 'SystemUtility' => `SystemUtility`